editor/editor_properties.cpp

```diff
@@ -2751,6 +2751,18 @@ void EditorPropertyNodePath::_node_assign() {
 		add_child(scene_tree);
 		scene_tree->connect("selected", callable_mp(this, &EditorPropertyNodePath::_node_selected));
 	}
+
+	scene_tree->get_scene_tree()->set_marked(get_base_node(), true, true);
+	Variant val = get_edited_property_value();
+	Node *n = nullptr;
+	if (val.get_type() == Variant::Type::NODE_PATH) {
+		NodePath path = val;
+		n = get_base_node()->get_node_or_null(path);
+	} else {
+		n = Object::cast_to<Node>(val);
+	}
```

Comment:
I assume the `path` here was for an explicit cast but I believe it may not be necessary for Variant?
```suggestion
	if (val.get_type() == Variant::Type::NODE_PATH) {
		n = get_base_node()->get_node_or_null(val);
	} else {
		n = Object::cast_to<Node>(val);
	}
```